clang/test/KeyInstructions/init-agg.cpp

```diff
@@ -0,0 +1,39 @@
+
+// RUN: %clang -gkey-instructions %s -gmlt -gno-column-info -S -emit-llvm -o - -ftrivial-auto-var-init=pattern \
+// RUN: | FileCheck %s --implicit-check-not atomGroup --implicit-check-not atomRank
+
+// The implicit-check-not is important; we don't want the GEPs created for the
+// store locations to be included in the atom group.
+
+int g;
+void a() {
+// CHECK: _Z1av()
+// CHECK: call void @llvm.memcpy{{.*}}, !dbg [[G1R1:!.*]]
```

Comment:
Hmmm -- this is ultimately checking that there /is/ a memcpy of group 1, but it could technically be for anything, including a partial initialization of something. Could we attempt to anchor it a bit more against the "A" variable please.